src/cli/commands/build/build.ts

```diff
@@ -37,9 +40,11 @@ export async function build(options: SWACLIConfig) {
     return;
   }
 
+  let projectConfig;
+  const detectedFolders = await detectProjectFolders(appLocation);
+  projectConfig = await generateConfiguration(detectedFolders.app[0], detectedFolders.api[0]);
```

Comment:
`  projectConfig = await generateConfiguration(detectedFolders.app[0], detectedFolders.api[0]);
`
why is it no longer needed to be part of try catch ?